.github/workflows/tox.yml

```
@@ -37,6 +37,7 @@ jobs:
             lint
             docs
             pkg
+            devel
```

```suggestion
            devel:tox -e py -- -k test_unit
```

maybe you want this?